### ISSUE TITLE
[Design] Accessibility checks required for staging review 

### ISSUE THREAD

**Issue by mshea0606** [department-of-veterans-affairs/va.gov-team#11279]
## Background

There are a few accessibility checks required as pre-requisites for the Staging Review. They are listed in the document below.

[508 pre-staging review checks](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#axe-scans-in-daily-development)

## Tasks
- [x]  [Color Tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#color-tests)
- [x] [Content resize check](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#content-resize-check)
- [x] [Keyboard navigation check ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#keyboard-navigation-check)

**Comment by mshea0606** [comment]
@tressaellen can you add an estimate in here when you get a chance?

**Comment by tressaellen** [comment]
Recording any issues found here:

Color Tests: Passed - the only potential issue could be the military badges.  Do I need to test each of those? (I've asked Jen Strickland, and she's getting back with me)

Content Resize Check: The only issue is the mobile navigation doesn't scroll on small screens.  (This is a known issue)

Keyboard Navigation Check: Passed

@mshea0606 how do you want me to proceed from here? Do I need to document this anywhere else?

**Comment by tressaellen** [comment]
Update - Jen says we don't need to worry about the badges.

**Comment by mshea0606** [comment]
@tressaellen  I think keeping issues documented in this ticket should be ok for now. cc:@jenstrickland

**Comment by jenstrickland** [comment]
I recommend adding the [checklist from here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/vsa-accessibility-test-cases-checklist.md) to your staging review request, and checking off the items as they are completed.
